runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1046,6 +1051,32 @@ TR::CompilationInfoPerThread::CompilationInfoPerThread(TR::CompilationInfo &comp
    _lastTimeThreadWasSuspended = 0;
    _lastTimeThreadWentToSleep = 0;
 
+#define FN_BUF_SIZE 1025
+#define COMP_ID_BUFF_SIZE 20
+
+   char *rtLogFileName = ((TR_JitPrivateConfig*)jitConfig->privateConfig)->rtLogFileName;
+   if (rtLogFileName)
+      {
+      char fn[FN_BUF_SIZE];
+      char compIdBuf[COMP_ID_BUFF_SIZE];
+
+
+      TR::snprintfNoTrunc(compIdBuf, COMP_ID_BUFF_SIZE, "%i", getCompThreadId());
+      bool truncated = TR::snprintfTrunc(fn, FN_BUF_SIZE, "%s.%s", rtLogFileName, compIdBuf);
+
+      if (!truncated)
+         _rtLogFile = fileOpen(TR::Options::getCmdLineOptions(), jitConfig, fn, "wb", true);
```

Comment:
Addressed in https://github.com/eclipse-openj9/openj9/pull/14210/commits/0eaf52c7e9c49fc3944eeddaa337812921d90c07. I just used the AOT option since the rtLog should really only be specified on the AOT cmd line.

Reply:
That makes sense to me :+1:, but did you mean to update this line? I thought it would now call `getAOTCmdLineOptions()`

Reply:
I thought I did :S...I must be losing my mind heh.

Reply:
Ok now actually addressed in https://github.com/eclipse-openj9/openj9/pull/14210/commits/d9685a1614d409fa1feb4cdd3dea0c8adc711e82 (see force push https://github.com/eclipse-openj9/openj9/compare/cdd950d39a241eee61c7f72efe86a35268fd6d56..60dc5a002c4c3b21f78a9b193d0e67ebb86f0034)